docs/introduction/sla.md

```diff
@@ -0,0 +1,222 @@
+customer, Rahti Container Cloud Service and the service provider, CSC - IT
+Center for Science Ltd., to cover the provision and support of the service as
+described hereafter. Amendments, comments and suggestions must be addressed
+using the communication channels defined in the section [Communication,
+reporting and escalation](#communication-reporting-and-escalation). The
+service provider retains right to introduce changes to the infrastructure. If
+the Customer does not accept the changes, this service subscription can be
+terminated.
+
+## Scope and description of the service
+
+The Rahti container cloud service is a cloud computing service. The Rahti
+service is based on OKD, which is a distribution of Kubernetes. The Rahti
+service allows its customers or users to run their own applications on top of
+it, often implemented as containers. The Rahti service provides, but is not
+necessarily limited to, these resources: 
+
+ * Hosting of container instances, and groups of container instances
+ * Storage services 
+ * Virtual networks for connecting container instances
+ * Load balancing of traffic to user workloads
+ * Features for replication, rolling updates, auto-recovery and auto-scaling of user workloads
+ * Basic default domain name and TLS for hosted web applications (under rahtiapp.fi)
+
+Customers can manage their resources using a web interface accessible through a
+web browser and through a set of APIs which allow programmatic management of
+resources. In order to access and use the service the customer must have a CSC
+user account.
+
+The user's applications are separated from other users’ applications from a
+network, storage and computational view.
+
+The Rahti service accounts for service usage based on the resources used. Up to
+date resource cost information is found in the [Rahti documentation](https://rahti.csc.fi)
+
+The normal functioning of the service is defined as
+
+ * the ability to launch new user applications.
+ * the availablility and integrity of the storage services.
+ * the functioning of the network, and the accessibility of user applications.
+ * the functioning of replication, auto-recovery and auto-scaling of user applications.
+
+This service is provided under User policy and terms of usage for the Rahti
+contariner cloud service hence by agreeing to this document the customer also
+agrees with CSC General Terms of Use and CSC’s Security Policy.
+
+## Service hours & exceptions
+
+The service is designed to run continuously. However, the following exceptions apply:
+
+Scheduled maintenances:
+
+ * Planned service breaks - Announced to the customer at least three weeks in advance
+ * Unplanned service breaks
+
+## Service components & dependencies
+
+The Rahti service is depends on the following services. The dependencies of these services are described in their respective service documentation.
+
+ * The cPouta service
+ * CSC user administration
```

Comment:
This is "CSC’s identity management service" in the cPouta SLA.

Reply:
Correct. However I think CSC user management might describe the dependency better, since it also depends on all the processes.

Reply:
Should we also link to the documentation here? The text says that "The dependencies of these services are described in their respective service documentation.", but as a prospective user of Rahti, would I know where to find this documentation without links? Would I be able to find a document that describes a service called "CSC user administration"? If that document exists, would it be easily identifiable as the service documentation for "CSC user administration", or would it have a different title?

It might be that not all of the dependencies have such documentation, or it would not be relevant. For example, the link for the service desk could simply be https://research.csc.fi/support that has a contact form and information about the service desk.

Listing "Kajaani datacenter infrastructure" and "Funet network" here explicitly could also be helpful, as customers might not otherwise realize that Rahti is dependent on them. This could be relevant in a situation where Rahti is unavailable due to a problem in the Funet network, for example. While cPouta has these in its SLA already and cPouta is listed here, the connection may not be obvious to end users.

Reply:
Added the other services as references, but removed the statement of their dependency lists. I don't see that we'd like to maintain that list in the Rahti SLA